tests/unit/test_keychain.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+# pylint: disable=duplicate-code,missing-function-docstring
+
+"""Chrony unit tests."""
+
+
+def test_keychain_private_key(mock_tls_keychain):
```

Comment:
Would you be able to add arrange/assert/act here please? :D

Reply:
Sorry I forget this part, added, thanks!